app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,21 @@
+class CommentsController < ApplicationController
+
+  def create 
+    @job = Job.find(params[:job_id])
+    @comment = Comment.new(comment_params)
+    @comment.job_id = @job.id
```

Comment:
Thanks for this Casey! I was making an association the other day and used this way, so I'm glad to know which one is preferable.